### ISSUE TITLE
🐛 Extra newlines are printed in `<script>` tags when rule fixes are applied on `next`

### ISSUE THREAD

**Issue by dyc3** [biomejs/biome#9050]
### Environment information

<details>

```bash

playground

```

</details>

### What happened?

https://next.biomejs.dev/playground/?code=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiACAAcwBlAHQAdQBwAD4ACgBpAG0AcABvAHIAdAAgAHsAIABjAG8AbQBwAHUAdABlAGQAIAB9ACAAZgByAG8AbQAgACIAdgB1AGUAIgA7AAoAPAAvAHMAYwByAGkAcAB0AD4ACgAKAA%3D%3D&language=vue

See that formatting does not result in extra newlines, but the analyzer fixes tab does. Only occurs on `next`.

### Expected result

It should not add the extra newlines around the embedded snippet when applying lint fixes.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
